zproject/urls.py

```diff
@@ -166,6 +166,7 @@
     url(r'^api/v1/external/updown$',        'webhooks.updown.api_updown_webhook'),
     url(r'^api/v1/external/yo$',            'webhooks.yo.api_yo_app_webhook'),
     url(r'^api/v1/external/zendesk$',       'webhooks.zendesk.api_zendesk_webhook'),
+    url(r'^api/v1/external/trello',         'webhooks.trello.api_trello_webhook'),
```

Comment:
can you put this in the right place alphabetically (and fix the missing $, preserving the alignment?).